var/spack/repos/builtin/packages/py-colorspacious/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColorspacious(PythonPackage):
+    """`colorlog.ColoredFormatter` is a formatter for use with Python's
+    `logging` module that outputs records using terminal colors.
+     """
+
+    homepage = "https://github.com/borntyping/python-colorlog"
+    url      = "https://pypi.io/packages/source/c/colorspacious/colorspacious-1.1.2.tar.gz"
+
+    version('1.1.2', sha256='5e9072e8cdca889dac445c35c9362a22ccf758e97b00b79ff0d5a7ba3e11b618')
+
+    depends_on('py-numpy')
```

Comment:
Can you double check dependencies? `py-numpy` doesn't seem to be listed in `setup.py`.

Reply:
I checked the master branch and the 1.1.2 version and `numpy` is in the *install_requires* : [current_setup](https://github.com/njsmith/colorspacious/blob/58948923b706879a54071568c7501be3f108797c/setup.py#L35).

Reply:
@Bidibulke I think I got the issue. I check the [homepage](https://github.com/borntyping/python-colorlog) currently in the package and it doesn't match the source being retrieved. Can you fix that?

Reply:
The link in your previous comment is for `colorlog`, not `colorspacious`.